src/pages/RemoveLiquidity/V3.tsx

```diff
@@ -320,7 +320,8 @@ function Remove({ tokenId }: { tokenId: BigNumber }) {
                   <ThemedText.DeprecatedLabel
                     ml="10px"
                     fontSize="20px"
-                  >{`${feeValue0?.currency?.symbol}/${feeValue1?.currency?.symbol}`}</ThemedText.DeprecatedLabel>
+                    id="remove-liquidity-tokens"
+                  >{`${liquidityValue0?.currency?.symbol}/${liquidityValue1?.currency?.symbol}`}</ThemedText.DeprecatedLabel>
```

Comment:
seems like feeValue hook is flaky in cypress? but feeValue0.currency should be the same as liquidityValue0.currency

Reply:
what issue were we seeing? I'm hesitant to change working functionality to pass a test

Reply:
traced it back to this open TODO in useV3PositionFees: https://github.com/Uniswap/interface/blob/5c53d8237d3dfd1e23d192ac499df902503d60a4/src/hooks/useV3PositionFees.ts#L33

may not explicitly be a cypress issue but am seeing this call fail -> catch -> return undefined for `feeValue`

Reply:
Interesting, okay. I'm fine with this change but can we also change the `DoubleCurrencyLogo` to use `liquidityValue` for consistency?

```
               <DoubleCurrencyLogo
                    currency0={liquidityValue0?.currency}
                    currency1={liquidityValue1?.currency}
                    size={20}
                    margin={true}
                  />

Reply:
sounds good good catch!